build/oss/dockerfiles/ubuntu-1804

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:18.04
+LABEL maintainers="Alibaba Cloud Authors"
+LABEL description="Alibaba Cloud CSI Plugin"
+
+ARG ossfsVer=1.80.6
+COPY ossfs_1.80.6_ubuntu18.04_amd64.deb /root/ossfs_1.80.6_ubuntu18.04_amd64.deb
+# RUN curl https://ack-csiplugin.oss-cn-hangzhou.aliyuncs.com/ossfs/ossfs_${ossfsVer}_centos7.0_x86_64.rpm -o /root/ossfs_${ossfsVer}_centos7.0_x86_64.rpm
```

Comment:
correct this url address to the right one, it should be a ubuntu specific version.